test/testdata/rewriter/minitest_tables.rb

```diff
@@ -0,0 +1,64 @@
+# typed: true
+
+class A
+  def foo; end
+end
+class B < A
+end
```

Comment:
I was confused to see `type: A` below instead of `T.any(A, B)`. Can you rename these to `Parent` / `Child`?